dynatrace_topology/stackstate_checks/dynatrace_topology/dynatrace_topology.py

```diff
@@ -283,6 +298,8 @@ def _collect_relations(self, dynatrace_component, external_id, component_type):
                     if component_type == 'custom-device':
                         target_relation_id = target_id.get('id')
                         self.relation(external_id, target_relation_id, relation_type, {})
+                    elif component_type == 'monitor':
+                        self.relation(target_id, external_id, relation_type, {})
```

Comment:
We don't have an example to get rest api response data for relation from monitor component to some other component. We should try to create that scenario on our Dynatrace trail instance, so we can write a unit test and check if generated relation is OK.